lib/eex/lib/eex/engine.ex

```diff
@@ -98,21 +101,24 @@ defmodule EEx.Engine do
   @doc """
   Returns an empty string as initial buffer.
   """
-  def init(_opts) do
+  @spec init(Keyword.t) :: <<>>
+  def init(opts) when is_list(opts) do
```

Comment:
> ExDoc already mentions if a callback has no documentation.
@josevalim I just checked, and if callback function has no doc, it doesn't show anything.

> maybe we could be more explicit and explicitly link to the callback in the documentation.
I will submit a another PR for that